pkg/apiserver/slowquery/queries.go

```diff
@@ -93,13 +93,13 @@ type SlowQuery struct {
 }
 
 type GetListRequest struct {
-	RangeBeginTs uint     `json:"rangeBeginTs" form:"rangeBeginTs"`
-	RangeEndTs   uint     `json:"rangeEndTs" form:"rangeEndTs"`
-	DB           []string `json:"db" form:"db"`
-	Limit        uint     `json:"limit" form:"limit"`
-	Text         string   `json:"text" form:"text"`
-	OrderBy      string   `json:"orderBy" form:"orderBy"`
-	IsDesc       bool     `json:"desc" form:"desc"`
+	BeginTime int      `json:"begin_time" form:"begin_time"`
+	EndTime   int      `json:"end_time" form:"end_time"`
+	DB        []string `json:"db" form:"db"`
+	Limit     int      `json:"limit" form:"limit"`
```

Comment:
I think uint is better for limit. Limit can't be negative.

Reply:
Right. In current implementation on `master` branch, limit < 0 means no limit. In this PR there is no need for limit < 0 anymore, limit = 0 means no limit instead.